### PR TITLE
Add stealth to tests

### PR DESCRIPTION
**Pull request by TinyGambe**
# Fix e2e Tests
Added missing scripts

Screenshot 2024-11-14 at 10 50 55 AM

### CODE REVIEW

**Review by paveldudka — APPROVED** [review]

File: tests/e2e/test_all_scripts.py [hunk 1]
```diff
@@ -3,6 +3,10 @@
 import logging
 from playwright.sync_api import sync_playwright, Page as SyncPage
 from playwright.async_api import async_playwright, Page as AsyncPage
+from playwright_stealth import stealth_sync, stealth_async
```
Comment (Zechereh) [inline tests/e2e/test_all_scripts.py:6]: Remove unnecessary imports

File: tests/e2e/test_all_scripts.py [hunk 2]
```diff
@@ -53,14 +57,33 @@ def get_test_id(config: ScriptConfig) -> str:
...
 
 
+def configs(page: SyncPage):
+
+    utils_script = from_file("utils.js")
+    magic_arrays_script = from_file("generate.magic.arrays.js")
+    page.add_init_script(utils_script)
+    page.add_init_script(magic_arrays_script)
+
+
```
Comment (Zechereh) [inline tests/e2e/test_all_scripts.py:67]: Is this function used anywhere?

File: tests/e2e/test_all_scripts.py [hunk 3]
```diff
@@ -53,14 +57,33 @@ def get_test_id(config: ScriptConfig) -> str:
...
+        properties = Properties()
+        opts = json.dumps(properties.as_dict())
+        opts = f"const opts = {opts}"
+
+        combined_script = (
+            opts + "\n" + utils_script + "\n" + magic_arrays_script + "\n" + config.script
+        )
+
+        page.add_init_script(combined_script)
```
Comment (Zechereh) [inline tests/e2e/test_all_scripts.py:86]: Looks legit

**Review by Zechereh — APPROVED** [review]
A few nits. But otherwise, thanks for the fix Gambe!

**Pull request merged**